examples/dogleg/run_dogleg_reverse.py

```diff
@@ -40,8 +40,8 @@
         alpha_x=1.3385830279518021,
         alpha_y=-1.3479109197361046,
         alpha_t=92.624347459169869,
+        dispX=-0.26669723385388505,
```

Comment:
Small break, just introduced in #1099 
```suggestion
        dispersion_x=-0.26669723385388505,
```